src/compiler/checker.ts

```diff
@@ -35043,13 +35048,13 @@ namespace ts {
                             // Include the `<>` in the error message
                             : rangeOfTypeParameters(sourceFile, parent.typeParameters!);
                         const only = parent.typeParameters!.length === 1;
-                        const message = only ? Diagnostics._0_is_declared_but_its_value_is_never_read : Diagnostics.All_type_parameters_are_unused;
+                        const message = only ? Diagnostics._0_is_declared_but_its_value_is_never_read : Diagnostics.All_type_parameters_are_unused; //TODO: 41974, unusedTypeParameters_TemplateTag

```

Comment:
Either remove the TODO if appropriate or move it to the line above so it's easier to see